x/emissions/keeper/keeper.go

```diff
@@ -484,68 +484,76 @@ func (k *Keeper) SetOneInForecasterSelfNetworkRegret(ctx context.Context, topicI
 
 // Returns the regret of a worker from comparing loss of worker relative to loss of other inferers
 // Returns (0, true) if no regret is found
-func (k *Keeper) GetInfererNetworkRegret(ctx context.Context, topicId TopicId, worker ActorId) (types.TimestampedValue, bool, error) {
+func (k *Keeper) GetInfererNetworkRegret(ctx context.Context, topicId TopicId, worker ActorId) (types.TimestampedValue, error) {
 	key := collections.Join(topicId, worker)
 	regret, err := k.latestInfererNetworkRegrets.Get(ctx, key)
 	if err != nil {
 		if errors.Is(err, collections.ErrNotFound) {
+			topic, err := k.GetTopic(ctx, topicId)
+			if err != nil {
+				return types.TimestampedValue{}, err
+			}
 			return types.TimestampedValue{
 				BlockHeight: 0,
-				Value:       alloraMath.NewDecFromInt64(0),
-			}, true, nil
+				Value:       topic.InitialRegret,
+			}, nil
 		}
-		return types.TimestampedValue{}, false, err
+		return types.TimestampedValue{}, err
 	}
-	return regret, false, nil
+	return regret, nil
 }
 
 // Returns the regret of a worker from comparing loss of worker relative to loss of other inferers
 // Returns (0, true) if no regret is found
-func (k *Keeper) GetForecasterNetworkRegret(ctx context.Context, topicId TopicId, worker ActorId) (types.TimestampedValue, bool, error) {
+func (k *Keeper) GetForecasterNetworkRegret(ctx context.Context, topicId TopicId, worker ActorId) (types.TimestampedValue, error) {
 	key := collections.Join(topicId, worker)
 	regret, err := k.latestForecasterNetworkRegrets.Get(ctx, key)
 	if err != nil {
 		if errors.Is(err, collections.ErrNotFound) {
+			topic, err := k.GetTopic(ctx, topicId)
+			if err != nil {
+				return types.TimestampedValue{}, err
+			}
 			return types.TimestampedValue{
 				BlockHeight: 0,
-				Value:       alloraMath.NewDecFromInt64(0),
-			}, true, nil
+				Value:       topic.InitialRegret,
+			}, nil
 		}
-		return types.TimestampedValue{}, false, err
+		return types.TimestampedValue{}, err
 	}
-	return regret, false, nil
+	return regret, nil
 }
 
 // Returns the regret of a worker from comparing loss of worker relative to loss of other inferers
 // Returns (0, true) if no regret is found
-func (k *Keeper) GetOneInForecasterNetworkRegret(ctx context.Context, topicId TopicId, forecaster ActorId, inferer ActorId) (types.TimestampedValue, bool, error) {
+func (k *Keeper) GetOneInForecasterNetworkRegret(ctx context.Context, topicId TopicId, forecaster ActorId, inferer ActorId) (types.TimestampedValue, error) {
 	key := collections.Join3(topicId, forecaster, inferer)
 	regret, err := k.latestOneInForecasterNetworkRegrets.Get(ctx, key)
 	if err != nil {
 		if errors.Is(err, collections.ErrNotFound) {
 			return types.TimestampedValue{
 				BlockHeight: 0,
 				Value:       alloraMath.NewDecFromInt64(0),
```

Comment:
Shouldn't this also be updated to be InitialRegret?

Reply:
done!